apps/dashboard/src/modules/admin/views/AdminSingleUserView.vue

```diff
@@ -1,22 +1,21 @@
 <template>
   <div class="page-container">
-    <div class="page-title">{{ `${currentUser ? currentUser.firstName : ''}'s profile` }}</div>
-
+    <div class="page-title">
+      {{ t('modules.admin.singleUser.profile', {user: currentUser ? currentUser.firstName : ''}) }}
```

Comment:
```suggestion
      {{ t('modules.admin.singleUser.profile', {user: currentUser ?? currentUser.firstName}) }}
```

Reply:
This does not work as it then interprets the type of currentUser as `never`